src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -921,17 +922,27 @@ private IEnumerable<FullBackup.StorageEnvironmentInformation> GetAllStoragesForB
 
                             smugglerResult = smuggler.Execute();
                         }
+
+                        switch (outputStream)
+                        {
+                            case EncryptingXChaCha20Poly1305Stream encryptedStream:
+                                encryptedStream.Flush(flushToDisk: true);
+                                break;
+                            case Stream fileStream:
+                                fileStream.Flush();
```

Comment:
Also `flushToDisk: true` if the `outputStream` is a `FileStream`

Reply:
FileStream is a stream. changed the name in https://github.com/ravendb/ravendb/pull/8506